### ISSUE TITLE
need a slash symbol in index.js

### ISSUE THREAD

**Issue by macdoor** [seriousme/fastify-openapi-glue#138]
index.js in generated project

`  service: `${__dirname}service.js`,
`
should be
`  service: `${__dirname}/service.js`,
`

index.js
```
// Fastify plugin autogenerated by fastify-openapi-glue
const openapiGlue = require("fastify-openapi-glue");

const options = {
  specification: `${__dirname}/openApi.json`,
  service: `${__dirname}service.js`,
  securityHandlers: `${__dirname}security.js`
};

module.exports = async function(fastify, opts) {
  fastify.register(openapiGlue, options);
};

```

**Comment by seriousme** [comment]
Hi, 

thanks for reporting this issue but I have some trouble trying to reproduce the error.

The example [index.js](https://github.com/seriousme/fastify-openapi-glue/blob/master/examples/generatedProject/index.js) is automatically generated during testing and contains the slash before service.js

I did find another error though:

```javascript
securityHandlers: `${__dirname}security.js`
```

is missing the slash:
```javascript
securityHandlers: `${__dirname}/security.js`
```

I just fixed this in #139 

Could you please check if this solves your problem ?

Kind regards,
Hans

**Comment by macdoor** [comment]
It's okay now

**Comment by seriousme** [comment]
Thx

Just published 2.2.1 on NPM

Kind regards,
Hans
